koji/collection-mash-split.py

```diff
@@ -125,12 +123,13 @@ def createrepo(self, path, comps=None, checksum=None):
 
         @param path: path to repository
         @type path: str
-        @param comps: comps file (not mandatory)
+        @param comps: comps file content (not mandatory)
         @type comps: str
         """
         cmd = "createrepo --pretty --quiet --database "
         if comps:
-            cmd += "--groupfile %s " % comps
+            with tempfile.NamedTemporaryFile() as comps_file:
```

Comment:
```suggestion
            with tempfile.NamedTemporaryFile() as comps_file:
                comps_file.write(comps)
```

Reply:
and probably .close() so that it's fully written?

Reply:
I just called `flush()` instead.